packages/backend/server/src/plugins/payment/quota.ts

```diff
@@ -31,6 +33,7 @@ export class TeamQuotaOverride {
           { memberLimit: quantity }
         );
         await this.permission.refreshSeatStatus(workspaceId, quantity);
+        await this.workspace.sendTeamWorkspaceUpgradedEmail(workspaceId);
```

Comment:
every time the subscription been updated, including quantity, will trigger activated event

Reply:
need to separate `workspace.subscription.created` and `workspace.subscription.updated`